src/components/Budget/AddBudgetModal/AddBudgetModal.js

```diff
@@ -29,7 +29,7 @@ const AddBudgetModal = ({ history, mutator, resources, onClose, fund, budgetStat
         const budget = await mutator.budget.POST({
           ...formValue,
           fundId: fund.id,
-          name: `${fund.code}-${fiscalYear.code}`,
+          name: `${fund.code}-${fiscalYear.label}`,
```

Comment:
so fiscalYear doesn't have `code` field anymore?

Reply:
It has, but here `fiscalYear.code` = `fiscalYear.label`. `fiscalYear.code` is required. See https://github.com/folio-org/ui-finance/pull/156/files#diff-4337fea6055306941a9b97d7f6896821R21 and https://github.com/folio-org/ui-finance/blob/master/src/Utils/getFiscalYearsForSelect.js#L5